_includes/multi-sources.html

```diff
@@ -0,0 +1,27 @@
+{% assign tag = include.tag %}
+{% assign src = include.src %}
+{% assign additional_types = include.additional_types %}
+{% assign supported_extensions = include.supported_extensions %}
+
+{% assign extensions = supported_extensions | split: '|' %}
+{% assign path_struct = src | split: '.' %}
+{% assign base_path = path_struct.first %}
```

Comment:
This does not work for a path containing a dot and so may raise questions. Do you think it worse it to fix?

```diff
- {% assign extensions = supported_extensions | split: '|' %}
- {% assign path_struct = src | split: '.' %}
- {% assign base_path = path_struct.first %}
+ {% assign base_path = src | split: '.' | pop | join: '.' %}
+ {% assign original_media_type = src | split: '.' | last | downcase %}
```

Reply:
I rewrote the implementation and now the code supports even this path format `relative.path/to/my.audio/ai.audio.mp3` and will correctly split the path into the necessary parts